app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    def __init__(self, message: str = "All friends should be vaccinated") -> None:
+        super().__init__(message)
+
+class NotVaccinatedError(VaccineError):
+    pass

```

Comment:
Consider providing a default message for `NotVaccinatedError` by overriding the constructor. This will make the error more descriptive when raised without a specific message.